src/main/java/frc/robot/RobotContainer.java

```diff
@@ -240,16 +249,15 @@ private void configureButtonBindings() {
 
     operatorController.b().whileTrue(orchestrator.expelIntakeIndex());
     operatorController.rightBumper().whileTrue(orchestrator.expelShindex());
-    operatorController.a().whileTrue(orchestrator.shootBasic());
-    operatorController.x().whileTrue(orchestrator.scoreAmp());
+    operatorController.a().whileTrue(shooter.manualShoot(0.85));
 
     // operator d pad
     operatorController.pov(0).whileTrue(arm.runPercent(0.2));
     operatorController.pov(180).whileTrue(arm.runPercent(-0.2));
     operatorController.pov(90).whileTrue(arm.runPercent(0));
 
     driverController.rightBumper().whileTrue(arm.toSetpoint(ArmConstants.STOW));
-    driverController.leftBumper().whileTrue(orchestrator.scoreAmp());
+    driverController.leftBumper().onTrue(orchestrator.alignArmAmp());
```

Comment:
Adam's PR #199 has this still calling scoreAmp. Why not score?

Reply:
they wanted it to not automatically score